addons/website_sale/views/product_product_add.xml

```diff
@@ -6,9 +6,19 @@
     <field name="model">product.product</field>
     <field name="arch" type="xml">
         <form js_class="website_new_content_form">
-            <group>
+            <group name="pricing">
                 <field name="website_url" invisible="1"/>
+                <field name="company_id" invisible="1"/>
+                <field name="currency_id" invisible='1'/>
+                <field name="cost_currency_id" invisible="1"/>
                 <field name="name" placeholder="e.g. Cheese Burger" string="Product Name"/>
+                <label for="list_price" class="mt-1"/>
+                <div name="pricing">
```

Comment:
Is the name actually relevant? Shouldn't it use class="o_row"?

Reply:
Well I changed to name to make it more explicit, but for the class, according to the spec, the 2 fields should one above the other, so the <mt-1> class is I think mandatory. But indeed, the <span class="ms-2"> was not necessary. Tell me if it is good for u !